frontend/components/common/Layout/Footer.tsx

```diff
@@ -286,32 +291,39 @@ function NewsletterForm({
                <FooterNewsletterEmailLabel>
                   Enter your email
                </FooterNewsletterEmailLabel>
-               <FooterNewsletterEmailInput
-                  ref={inputRef}
-                  disabled={subscription.status !== SubscriptionStatus.Idle}
-                  placeholder={emailPlaceholder}
-               />
+               <HStack>
+                  <FooterNewsletterEmailInput
+                     ref={inputRef}
+                     disabled={subscription.status !== SubscriptionStatus.Idle}
```

Comment:
There are a few places in here where we could still switch to using isSubscribed.

Reply:
Or actually maybe not for some of these, since we're checking for subscribing too.

Reply:
Yeah `isSubscribed` is only when `subscription.status === SubscriptionStatus.Subscribed` and all such occurrences have been replaced by `isSubscribed`